web-app/src/games/api.admin.games.controller.ts

```diff
@@ -21,8 +27,21 @@ export class ApiAdminGamesController {
     return game;
   }
 
-  @Get('')
-  async findMany(): Promise<Game[]> {
-    return this.gamesService.findMany();
+  @Get()
+  async findMany(
+    @Query('s', new JsonPipe<GamesSearchParams>())
+    searchParams?: GamesSearchParams,
+  ): Promise<Game[]> {
+    return this.gamesService.findMany(searchParams);
```

Comment:
We need to return the total nb of entries in db to make the pagination work in the react admin ui !

Reply:
Indeed!

This method now returns an object of the form: 

```
{ 
  data: Game[]; 
  total: number 
}
```